pupil-spa/test/README.md

```diff
@@ -0,0 +1,98 @@
+Installing and Running MTC tests
+================================
+
+This is a short guide to installing and running the MTC tests.  
+
+##Clone the tests
+
+```bash
+git clone https://agilefactory.visualstudio.com/DefaultCollection/MTC/_git/app
+```
+
+##Install RVM stable with ruby
+
+Follow installation instructions for installing RVM here:
+
+https://rvm.io/rvm/install
+ 
+Once rvm is installed, we need ruby version 2.4.0, to install use the following:
+ `rvm install 2.4.0`
+ 
+Use ruby version 2.4.0 and set it as your default:
+ `rvm use 2.4.0 --default`
+
+##Installing required gems
+
+Go to the folder /test and run the following commands:
+
+`gem install bundler` and then `bundle install`
+
+These two commands will install all the gems that the tests need.
+
+## Running the tests
+
+#####Basic
+
+To run the tests with the default options ( headless and app running on localhost:3000 ) give the command below:
+
```

Comment:
when i run `gem install capybara` i get...

```
Building native extensions.  This could take a while...
ERROR:  Error installing capybara:
	ERROR: Failed to build gem native extension.

    current directory: /Users/guy/.rvm/gems/ruby-2.4.0/gems/nokogiri-1.8.0/ext/nokogiri
/Users/guy/.rvm/rubies/ruby-2.4.0/bin/ruby -r ./siteconf20170811-77518-1pe7pnz.rb extconf.rb
checking if the C compiler accepts ... yes
checking if the C compiler accepts -Wno-error=unused-command-line-argument-hard-error-in-future... no
Building nokogiri using packaged libraries.
Using mini_portile version 2.2.0
checking for iconv.h... yes
checking for gzdopen() in -lz... yes
checking for iconv using --with-opt-* flags... yes
...
```